code_schemes/yes_no.json

```diff
@@ -0,0 +1,92 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Yes_No",
+    "SchemeID": "Scheme-049y67c27cdcb",
+    "Codes": [{
+        "DisplayText": "Yes",
+        "VisibleInCoda": true,
+        "NumericValue": 0,
+        "CodeID": "code-ehg2ad7ce2cf",
+        "CodeType": "Normal",
+        "MatchValues": [
+          "yes"
+        ],
+        "StringValue": "yes"
+      },
+      {
+        "NumericValue": 1,
+        "CodeID": "code-41f5jk8a71ef",
```

Comment:
slightly weird that these ids are twice the length of the ids used in some of the other schemes